src/v/raft/tests/raft_fixture.cc

```diff
@@ -489,6 +490,8 @@ ss::future<> raft_node_instance::stop() {
         vlog(_logger.debug, "stopping protocol");
         co_await _buffered_protocol->stop();
         co_await _protocol->stop();
+        // release f_log pointer before stopping raft
```

Comment:
Any reason we do it before? It should not matter, as it's a shared pointer, so the underlying object will only die when `_raft` is gone.
Same question for stopping log before deleting raft, but it's out of scope of this PR.